api/ical_api.py

```diff
@@ -100,11 +100,7 @@ class ReservationUnitCalendarUrlViewSet(
 ):
     serializer_class = ReservationUnitCalendarUrlSerializer
     queryset = ReservationUnit.objects.all().prefetch_related("unit")
-    permission_classes = (
-        [ReservationUnitCalendarUrlPermission]
-        if not settings.TMP_PERMISSIONS_DISABLED
-        else [permissions.AllowAny]
-    )
+    permission_classes = [ReservationUnitCalendarUrlPermission]
```

Comment:
I guess its developer's choice when it was introduced. Since it can be list or tuple. The latter is immutable but easier to typo (if don't use the comma in one cell tuple it will get interpret to something else than tuple) => nevertheless we seem to use tuples everywhere so I'll change these to tuples(?)

Reply:
oh, actually those lists are in DRF side of table, so let those be since we're probably ditching whole rest api soonish?